### ISSUE TITLE
Windows: control characters in terminal

### ISSUE THREAD

**Issue by gagarin55** [ethereumproject/go-ethereum#485]
When I run geth in cmd.exe on windows platform instead of colorful output I get something like:
```
←[2m2018-01-31 12:27:19←[0m Discover
                     1/25 peers
←[2m2018-01-31 12:28:20←[0m Discover
                     1/25 peers
←[2m2018-01-31 12:28:46←[0m Mined    #2076541               d2e5785f
 0 txs/mgas          1/25 peers
```

I've checked another ethereum client, it can somehow detects the fact that terminal doesn't support ASCII colors (or what it is) and just use plain output.

**Comment by tzdybal** [comment]
Use console from git client, it works like a charm ;)

**Comment by whilei** [comment]
Resolved with #523
